SDL/ModuleConnectionMap.cc

```diff
@@ -1,27 +1,9 @@
 #include "ModuleConnectionMap.h"
 
 SDL::ModuleConnectionMap SDL::moduleConnectionMap;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer1Subdet5;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer2Subdet5;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer3Subdet5;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer1Subdet4;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer2Subdet4;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer3Subdet4;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer4Subdet4;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer1Subdet5_pos;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer2Subdet5_pos;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer3Subdet5_pos;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer1Subdet4_pos;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer2Subdet4_pos;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer3Subdet4_pos;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer4Subdet4_pos;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer1Subdet5_neg;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer2Subdet5_neg;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer3Subdet5_neg;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer1Subdet4_neg;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer2Subdet4_neg;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer3Subdet4_neg;
-SDL::ModuleConnectionMap SDL::moduleConnectionMap_pLStoLayer4Subdet4_neg;
+std::vector<SDL::ModuleConnectionMap> SDL::moduleConnectionMap_pLStoLayer(7);
```

Comment:
> if the size is well defined, `std::array` is better

@VourMa Did you ever address this comment?

Reply:
No, and I don't intend to do it now. See my last comment:
> I went with the least possible changes for now, since the map loading would have to be revisited soon.